core/field.js

```diff
@@ -619,10 +629,12 @@ Blockly.Field.prototype.showEditor = function(opt_e) {
  * check the size of the field.
  */
 Blockly.Field.prototype.updateWidth = function() {
-  console.warn('Deprecated call to updateWidth, call' +
-    ' Blockly.Field.updateSize_ to force an update to the size of the' +
-    ' field, or Blockly.utils.dom.getTextWidth() to check the size' +
-    ' of the field.');
+
+  Blockly.utils.deprecation.warn(
+      'Field.prototype.updateWidth',
+      'May 2019',
+      'December 2020',
+      'Blockly.Field.updateSize_ or Blockly.utils.dom.getTextWidth()');
```

Comment:
To be consistent with other recommendations, `Blockly.Field.prototype.updateSize_`

Reply:
and remove the parentheses around `Blockly.utils.dom.getTextWidth`